### ISSUE TITLE
Python 3.7 support

### ISSUE THREAD

**Issue by ppawiggers** [census-instrumentation/opencensus-python#348]
opencensus-python does not support Python 3.7, for example it's using a forbidden name (since Python 3.7):

```
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    from opencensus.stats.exporters import stackdriver_exporter as stackdriver
  File "/Users/peter/.virtualenvs/octest/lib/python3.7/site-packages/opencensus/stats/exporters/stackdriver_exporter.py", line 23
    from opencensus.common.transports import async
                                                 ^
SyntaxError: invalid syntax

**Comment by zoidbergwill** [comment]
@ppawiggers Can you do `from opencensus.common.transports.async import AsyncTransport`?

**Comment by zoidbergwill** [comment]
PRs are always welcome too.

We could potentially do something like rename it to a better name and still export it as `async` on Python 3.6 and earlier, and just use a better name going forward.

**Comment by c24t** [comment]
Fixed in #362.
